virttest/embedded_qemu.py

```diff
@@ -0,0 +1,105 @@
+"""
+Classes and functions for embedded qemu driver.
+"""
+import re
+import logging
+
+import aexpect
+from avocado.core import exceptions
+from avocado.utils import path
+from avocado.utils import process
+
+from virttest import utils_split_daemons
```

Comment:
I assume the module is  more specific to libvirt part, please put them into virttest/utils_libvirt/ folder

Reply:
Updated.